test/integration_test.py

```diff
@@ -714,3 +724,60 @@ def test_mini_dss_failures(self):
                 mini_dss.get_file(uuid, version, 'aws')
             with self.assertRaises(self.SpecialError):
                 mini_dss.get_native_file_url(uuid, version, 'aws')
+
+
+class AzulChaliceLocalIntegrationTest(AzulTestCase):
+    service_app = Path(config.project_root) / 'lambdas' / 'service'
+    host = '0.0.0.0'
```

Comment:
See if 127.0.0.1 works. I'd rather not listen on all interfaces, for security reasons.